src/multinode_run.jl

```diff
@@ -362,14 +356,15 @@ Return a Cyclades partitioning or an equal partitioning of the target sources.
 """
 function partition_box(npartitions::Int, target_sources::Vector{Int},
                        neighbor_map::Vector{Vector{Int}};
-                       cyclades=true,
+                       cyclades_partition=true,
                        batch_size=400)
-    if cyclades
+    if cyclades_partition
```

Comment:
This code is duplicated in `one_node_joint_infer`. Would you move this function to `joint_infer.jl` and call it from `one_node_joint_infer`, and remove the redundant code from `one_node_joint_infer`?